discovery-infra/start_discovery.py

```diff
@@ -288,7 +302,7 @@ def _cluster_create_params():
     ipv4 = args.ipv4 and args.ipv4.lower() in MachineNetwork.YES_VALUES
     ipv6 = args.ipv6 and args.ipv6.lower() in MachineNetwork.YES_VALUES
     ntp_source = _get_host_ip_from_cidr(args.vm_network_cidr6 if ipv6 and not ipv4 else args.vm_network_cidr)
-    user_managed_networking = is_none_platform_mode()
+    user_managed_networking = is_user_managed_networking()
```

Comment:
one of the ideas not to change this flag was to verify that there is no need to set it true in cluster create call

Reply:
It seemed to me that the definition of is_user_managed_networking here was incorrect.  It is not just for none platform but for single node as well